mne/__init__.py

```diff
@@ -98,6 +98,7 @@
 from . import viz
 from . import decoding
 from . import realtime
+from . import testing
```

Comment:
Yeah, the equivalent has been in there for a while. It would be good to see how it compares to this version.